run/settings.json

```diff
@@ -40,9 +40,14 @@
             "gapless":1,
             "max_cell_size":128,
             "use_size_limit":1,
-            "downsample":1
+            "downsample":0.7
```

Comment:
with this config, you downsample the textures by default, so what they do in ps and what they see ingame will be different